streams/src/main/resources/common/message/SubscriptionInfo.json

```diff
@@ -15,7 +15,7 @@
 
 {
   "name": "SubscriptionInfo",
-  "validVersions": "1-6",
+  "validVersions": "1-7",
```

Comment:
The KIP still states that we bump from 4 to 5. Could you please change that as well?

Reply:
Ack

Reply:
Clearly it's taken a while to get this KIP off the ground :P